dna/models/attention_regression_model.py

```diff
@@ -0,0 +1,44 @@
+from .base_models import RNNRegressionRankSubsetModelBase
+from .torch_modules.attention_mlp import AttentionMLP
+
+
+class AttentionRegressionModel(RNNRegressionRankSubsetModelBase):
+
+    def __init__(
+        self, n_layers: int, n_heads: int, attention_in_features: int, attention_hidden_features: int,
+        attention_activation_name: str, reduction: str, activation_name: str, dropout: float,
+        output_n_hidden_layers: int, output_hidden_layer_size: int, use_batch_norm: bool, use_skip: bool, *,
+        device: str = 'cuda:0', seed: int = 0
+    ):
+
+        super().__init__(activation_name, dropout, output_n_hidden_layers, output_hidden_layer_size, use_batch_norm,
+                         use_skip, device=device, seed=seed)
+
+        self.n_layers = n_layers
+        self.n_heads = n_heads
+        self.attention_in_features = attention_in_features
+        self.attention_hidden_features = attention_hidden_features
+        self.attention_activation_name = attention_activation_name
+        self.reduction = reduction
```

Comment:
let's call it `reduction_name` or something similar to indicate that it is a string and not the actual reduction funciton